skyportal/model_util.py

```diff
@@ -15,7 +15,7 @@
 
 all_acl_ids = ['Become user', 'Comment', 'Manage users', 'Manage sources',
                'Manage groups', 'Upload data', 'System admin', 'Post taxonomy',
-               'Delete taxonomy']
+               'Delete taxonomy', 'Classify']
 
 role_acls = {
```

Comment:
Should add "Classify" to relevant roles here

Reply:
@acrellin is making this an issue. Will address in the next PR.